habitat_code/kobylarz_kmeans_review_code_20240313.py

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env python
```

Comment:
I must say your Functions are impressive! I have had a tough time with this assignment, and so, in my eyes, your work looks great.   Everything was well documented, so I was able to break down what you were trying to do in each coding module very easily.